src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { SimpleMessageValues } from '~types/index';
```

Comment:
Usually we leave an empty line after outside imports & the dApp imports

Reply:
Fixed now, thank you olga